### PR TITLE
Provenance nested

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: cwltool/command_line_tool.py [hunk 1]
```diff
@@ -346,6 +346,8 @@ def rm_pending_output_callback(output_callbacks, jobcachepending,
                     # https://github.com/python/mypy/issues/797
                     partial(rm_pending_output_callback, output_callbacks,
                             jobcachepending))
+
+        builder = self._init_job(job_order, provObj, **kwargs)
```
Comment (mr-c) [inline cwltool/command_line_tool.py]: Twice? See below

Comment (FarahZKhan) [reply cwltool/command_line_tool.py]: fixed

File: cwltool/executors.py [hunk 2]
```diff
@@ -46,6 +52,7 @@ def run_jobs(self,
     def execute(self,
                 t,                 # type: Process
                 job_order_object,  # type: Dict[Text, Any]
+                make_fs_access=None,
```
Comment (mr-c) [inline cwltool/executors.py]: SyntaxError: duplicate argument 'make_fs_access' in function definition

Comment (FarahZKhan) [reply cwltool/executors.py]: fixed in 2022b3d

**Pull request merged**